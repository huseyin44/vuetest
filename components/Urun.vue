<template>
 <v-app>
  <div>
      <v-row>
          <v-col class="aragenis"></v-col>
          <v-col class="aragenis"></v-col>
          <v-col class="aragenis"></v-col>
          <v-col class="sutunbir">
            <v-row class="imagebir">
             <img style="max-height:108px;max-width:108px;" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_1.jpg">
            </v-row>
            <v-row class="arayuksek"></v-row>
            <v-row class="imagebir1">
             <img style="max-height:108px;max-width:108px;" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_2.jpg">
            </v-row>
          </v-col>
          <v-col class="aragenis"></v-col>
          <v-col >
            <v-row class="imageiki">
             <img class="imageuc" src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_1.jpg">
            </v-row>
          </v-col>
          <v-col>
              <v-row>
                <v-col>
                 <nuxt-link to="/"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Anasayfa
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col>
                 <nuxt-link to="/MarkaClick"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Saat
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col>
                 <nuxt-link to="/MarkaClick"  style="text-decoration: underline; color: inherit;white-space: nowrap;width:3%;">
                 Erkek Saat Modelleri
                 </nuxt-link>
                </v-col>
                <v-col style="width:0.1%">></v-col>
                <v-col><div style="color:gray;white-space: nowrap;width:3%;">AX2802 Erkek Kol Saati</div></v-col>
              </v-row>
              <v-row class="satir11">
                  <v-col>
                      <v-row>
                        <img class="markaresim" src="https://cdn.saatvesaat.com.tr/media/amshopby/big_1735e392dea76f18.png">
                      </v-row>
                      <v-row class="arayuksek"></v-row>
                      <v-row>
                          <nuxt-link to="/MarkaClick"  style=" color: inherit;">
                          Tüm Armani Exchange Ürünleri
                          </nuxt-link>
                      </v-row>
                  </v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col class="aragenis2"></v-col>
                  <v-col>
                      <div style="color:black;font-size:110%;">Türkiye Yetkili Satıcısı</div>
                  </v-col>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:black;font-size:130%;">Armani Exchange</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:150%;">AX2802 Erkek Kol Saati</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:100%;">Çelik / Quartz Teknoloji / 42 mm Kasa Çapı</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:gray;font-size:120%;text-decoration:line-through;">337000 TL</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:red;font-size:170%;">151600 TL</div>
              </v-row>
              <v-row class="arayuksek"></v-row>
              <v-row class="arayuksek"></v-row>
              <v-row class="arayuksek"></v-row>
              <v-row>
              <v-col style="padding-top:9%"><div>Adet</div></v-col>
              <v-col style="padding-top:9%">
                <select style="width:35%">
                 <option selected="selected" value="1">1</option>
                 <option value="2">2</option>
                 <option value="3">3</option>
                 <option value="4">4</option>
                 <option value="5">5</option>
                 <option value="6">6</option>
                 <option value="7">7</option>
                 <option value="8">8</option>
                 <option value="9">9</option>
                 <option value="10">10</option>
                </select>
              </v-col>
              <v-col>
              <nuxt-link to="/Alisverisdevam">
              <button style="text-decoration:none"  class="satinal">
                SATIN AL
              </button>
              </nuxt-link>
              </v-col>
              <v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col>
              <v-col class="aragenis2"></v-col><v-col class="aragenis2"></v-col>
              </v-row>
              <v-row>
                  <div style="color:353535;font-family:Roboto, Arial, sans-serif;">Web'e özel Ürünlerin iade ve değişim işlemleri sadece online kanaldan sağlanmakta olup, 
                      mağazalarımızda değişim gerçekleştirilmemektedir.</div>
              </v-row>
              <v-row class="arayuksek"></v-row><v-row class="arayuksek"></v-row>
              <v-row>
                <div style="color:lightgray;font-family:Roboto, Arial, sans-serif;">Sorularınız ve destek için online chat veya 0212 367 44 74 </div>
              </v-row>
          </v-col>
      </v-row>
  </div>
 </v-app>
</template>

<script>

</script>

<style>
.sutunbir{
 max-width: 110px;
}
.arayuksek{
 height:4px;
}
.aragenis{
 width:4px;
 max-width:4px ;
}
.aragenis2{
 width:24px;
 max-width:24px ;
}
.imagebir{
 display: inline-block;
 border: 1px solid #e4e4e4;
}
.imagebir1{
 display: inline-block;
 border: 1px solid #e4e4e4;
}
.imageiki{
 border: 1px solid #e4e4e4;
 max-width: 532px;
 height: 500px;
}
.imageuc{
 max-height: 480px;
 max-width: 520px;
}
.markaresim{
 max-height: 48px;
 max-width: 216px;
}
.satir11{
border-top-style:solid ;
border-bottom:solid ;
border-right:none ;
border-left:none ;
border-color:gray;
border-width: 1px;
width: 580px;
}
.satinal{
min-height: 40px;
line-height: 30px;
font-size: 28px;
padding: 15px 30px;
background: black;
color: white;
min-width: 390px;
width: 100%;
white-space: normal;
font-weight: 500;
border: 4px solid #000000;
text-align: center;
}
.satinal:hover{
 color: black;
  background: white;
  border-style:solid;
  border-color:black;
}
</style>