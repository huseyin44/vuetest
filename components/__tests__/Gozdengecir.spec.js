import { shallowMount,RouterLinkStub } from "@vue/test-utils";
import Urun from "../Gozdengecir";
import Vue from 'vue'
import Vuetify from 'vuetify'
Vue.use(Vuetify)
const factory = () => {
    return shallowMount(Urun, {
      stubs: {
        'nuxt-link': true,
        'any-other-child': true
       }
    });
  };
describe('Urun', () => {
    let vuetify
    beforeEach(() => {
      vuetify = new Vuetify()
    })
  
    it('Bir Vue kopmonenti', () => {
        const wrapper = factory();
        expect(wrapper.isVueInstance()).toBeTruthy();
    })
    it('Snapshot Ozelligi', () => {
        const wrapper = factory();
        expect(wrapper.html()).toMatchSnapshot();
    })
    test('triggers a mouseover', async () => {
        const wrapper = shallowMount(Urun)
        await wrapper.trigger('mouseover')
      })
    test('triggers a mouseleave', async () => {
        const wrapper = shallowMount(Urun)
        await wrapper.trigger('mouseleave')
      })
   
 }) 