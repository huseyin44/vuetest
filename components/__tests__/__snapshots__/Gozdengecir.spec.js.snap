// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Urun Snapshot Ozelligi 1`] = `
<div>
  <v-app-stub id="app" container="">
    <v-col-stub tag="div" class="border">
      <v-row-stub tag="div">
        <div style="padding: 0px 0px 0px 7px; font-size: 20px;">Siparis Gözden Geçir</div>
      </v-row-stub>
      <v-row-stub tag="div"></v-row-stub>
      <v-row-stub tag="div" class="dorduncu">
        <div style="color: black; padding: 0px 0px 0px 7px;">Sepet</div>
      </v-row-stub>
      <v-row-stub tag="div"></v-row-stub>
      <v-row-stub tag="div" class="ikincisatir">
        <v-col-stub tag="div"><img src="https://cdn.saatvesaat.com.tr/media/catalog/product/a/x/ax2802_1.jpg" style="max-height: 75px; max-width: 75px;"></v-col-stub>
        <v-col-stub tag="div">
          <div>Adet 1</div>
        </v-col-stub>
        <v-col-stub tag="div">
          <div style="padding: 28px 0px 0px 0px; color: gray;">337000 TL</div>
        </v-col-stub>
      </v-row-stub>
      <v-row-stub tag="div"></v-row-stub>
      <v-row-stub tag="div" class="ucuncusatir">
        <v-col-stub tag="div">
          <div style="padding: 0px 0px 0px 7px; color: rgb(119, 119, 119);">Ara Toplam</div>
        </v-col-stub>
        <v-col-stub tag="div">
          <v-row-stub tag="div">
            <div style="padding: 0px 0px 0px 70px; color: rgb(119, 119, 119);">337000 TL</div>
          </v-row-stub>
          <v-row-stub tag="div">
            <v-col-stub tag="div">İndirim</v-col-stub>
            <v-col-stub tag="div">-185400 TL</v-col-stub>
          </v-row-stub>
        </v-col-stub>
      </v-row-stub>
      <v-row-stub tag="div">
        <v-col-stub tag="div">
          <div style="padding: 0px 0px 0px 7px; color: rgb(156, 12, 12) !important; white-space: nowrap;">Genel Toplam</div>
        </v-col-stub>
        <v-col-stub tag="div">
          <div style="padding: 0px 0px 0px 60px;">151600 TL</div>
        </v-col-stub>
      </v-row-stub>
    </v-col-stub>
  </v-app-stub>
</div>
`;
